Panic when running `vt tester`

We are not configuring the loader correctly for `vt tester`, below the stack trace:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x18 pc=0x105b739c8]

goroutine 1 [running]:
github.com/vitessio/vt/go/tester.(*Tester).Run(0x140009401e0)
        /Users/florentpoinsard/Code/vitess-tester/go/tester/tester.go:227 +0x78
github.com/vitessio/vt/go/tester.ExecuteTests({0x14000429808, {{0x105b9273f, 0x9}, 0x1b05, {0x0, 0x0}, {0x0, 0x0}, {0x0, 0x0}, ...}, ...}, ...)
        /Users/florentpoinsard/Code/vitess-tester/go/tester/execute.go:57 +0x128
github.com/vitessio/vt/go/tester.Run({{0x105b7e5ed, 0x5}, 0x0, 0x0, 0x0, {0x16b88757e, 0x15}, {0x0, 0x0}, {0x0, ...}, ...})
        /Users/florentpoinsard/Code/vitess-tester/go/tester/run.go:120 +0x478
github.com/vitessio/vt/go/cmd.testerCmd.func1(0x14000502c08, {0x14000883cb0?, 0x4?, 0x105b7bc8a?})
        /Users/florentpoinsard/Code/vitess-tester/go/cmd/tester.go:39 +0x74
github.com/spf13/cobra.(*Command).execute(0x14000502c08, {0x14000883c20, 0x3, 0x3})
        /Users/florentpoinsard/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:985 +0x834
github.com/spf13/cobra.(*Command).ExecuteC(0x14000502608)
        /Users/florentpoinsard/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1117 +0x344
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/florentpoinsard/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1041
github.com/vitessio/vt/go/cmd.Execute()
        /Users/florentpoinsard/Code/vitess-tester/go/cmd/root.go:43 +0x10c
main.main()
        /Users/florentpoinsard/Code/vitess-tester/go/vt/main.go:24 +0x1c
```

--- a/go/cmd/tester.go
+++ b/go/cmd/tester.go
@@ -26,6 +26,7 @@ import (
 
 func testerCmd() *cobra.Command {
 	var cfg vttester.Config
+	var inputType string
 
 	cmd := &cobra.Command{
 		Aliases: []string{"test"},
@@ -36,6 +37,12 @@ func testerCmd() *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cfg.Tests = args
 			cfg.Compare = true
+			loader, err := configureLoader(inputType, true)
+			if err != nil {
+				return err
+			}
+			cfg.Loader = loader
+
 			return usageErr(cmd, vttester.Run(cfg))
 		},
 	}
@@ -44,6 +51,7 @@ func testerCmd() *cobra.Command {
 
 	cmd.Flags().BoolVar(&cfg.OLAP, "olap", false, "Use OLAP to run the queries.")
 	cmd.Flags().BoolVar(&cfg.XUnit, "xunit", false, "Get output in an xml file instead of errors directory")
+	addInputTypeFlag(cmd, &inputType)
 
 	return cmd
 }

